docs/data-ai/capture-data/advanced/advanced-data-capture-sync.md

```diff
@@ -0,0 +1,626 @@
+| `maximum_num_sync_threads` | int | Optional | Max number of CPU threads to use for syncing data to the Viam Cloud. <br> Default: [runtime.NumCPU](https://pkg.go.dev/runtime#NumCPU)/2 so half the number of logical CPUs available to viam-server |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
+| `mongo_capture_config.uri` | string | Optional | The [MongoDB URI](https://www.mongodb.com/docs/v6.2/reference/connection-string/) data capture will attempt to write tabular data to after it is enqueued to be written to disk. When non-empty, data capture will capture tabular data to the configured MongoDB database and collection at that URI.<br>See  `mongo_capture_config.database` and  `mongo_capture_config.collection` below for database and collection defaults.<br>See [Data capture directly to MongoDB](/data-ai/capture-data/advanced/how-sync-works/#storage) for an example config.|  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
+| `mongo_capture_config.database` | string | Optional | When `mongo_capture_config.uri` is non empty, changes the database data capture will write tabular data to. <br> Default: `"sensorData"`   |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
+| `mongo_capture_config.collection` | string | Optional | When `mongo_capture_config.uri` is non empty, changes the collection data capture will write tabular data to.<br> Default: `"readings"`   |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
+| `cache_size_kb` | float | Optional | `viam-micro-server` only. The maximum amount of storage bytes (in kilobytes) allocated to a data collector. <br> Default: `1` KB. |  <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
+| `file_last_modified_millis` | float | Optional | The amount of time to pass since arbitrary files were last modified until they are synced. Normal <file>.capture</file> files are synced as soon as they are able to be synced. <br> Default: `10000` milliseconds. |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
+
+{{< /expand >}}
+
+You can edit the JSON directly by switching to **JSON** mode in the UI.
+
+### Advanced data capture configurations
+
+{{< alert title="Caution" color="caution" >}}
+
+Avoid configuring data capture to higher rates than your hardware can handle, as this leads to performance degradation.
+
+{{< /alert >}}
+
+{{< tabs >}}
+{{% tab name="Regular" %}}
+
+{{< tabs >}}
+{{% tab name="viam-server" %}}
+
+This example configuration captures data from the `ReadImage` method of a camera:
+
+```json {class="line-numbers linkable-line-numbers"}
```

Comment:
[opt] Consider adding highlighted lines to all these long configs to identify the relevant parts